tests/local.mk

```diff
@@ -132,7 +132,7 @@ install-tests += $(foreach x, $(nix_tests), tests/$(x))
 
 clean-files += $(d)/tests/common/vars-and-functions.sh $(d)/config.nix $(d)/ca/config.nix
 
-test-deps += tests/common/vars-and-functions.sh tests/config.nix tests/ca/config.nix tests/plugins/libplugintest.$(SO_EXT)
+test-deps += tests/common/vars-and-functions.sh tests/config.nix tests/ca/config.nix
```

Comment:
Is that really intended?

Reply:
Ooops! I think a bad merge conflict fix I think.

Reply:
Actually it is not, check the next few lines: it was already added conditionally below.

We indeed should skip this in static builds as the `dlopen` won't work right with them anyways, right?

Reply:
Oooh indeed. Makes sense :+1: